PRASCore.jl/src/Systems/SystemModel.jl

```diff
@@ -152,3 +158,156 @@ function consistent_idxs(idxss::Vector{UnitRange{Int}}, nitems::Int, ngroups::In
     return true
 
 end
+
+function Base.show(io::IO, sys::SystemModel{N,L,T,P,E}) where {N,L,T<:Period,P<:PowerUnit,E<:EnergyUnit}
+    time_unit = unitsymbol_long(T)
+    print(io, "SystemModel($(length(sys.regions)) regions, $(length(sys.interfaces)) interfaces, ",
+          "$(length(sys.generators)) generators, $(length(sys.storages)) storages, ",
+          "$(length(sys.generatorstorages)) generator-storages,",
```

Comment:
Possibly Nitpicky (PNP) - can you remove hyphen from generator-storages when printing to be consistent with PRAS struct names?